Makefile

```diff
@@ -1,6 +1,8 @@
 CI_FOLDER = images
-PIPE_IMAGE ?= quay.io/ztpfw/pipeline
-UI_IMAGE = quay.io/ztpfw/ui
+#Default: PIPELINE_ACCOUNT = ztpfw
+PIPELINE_ACCOUNT = takinosh
```

Comment:
Why do you need your account here? you don't need it using the same strategic showed before...
You will build an image for the branch for testing purposes, use the pipeline with this branch image, and when it will be merged, it will be available as a contrib for the rest of people interested. We should be generic at this point ;)